Blog post promoting GopherCon 2019 liveblog (#152)
sourcegraph/about: blogposts/livebloggig-gophercon-2019.md

author: Ryan Blunden - Developer Advocate at Sourcegraph
publishDate: 2019-07-22T00:00-07:00
tags: [
  blog
]
slug: sourcegraph-liveblogging-at-gophercon-2019
heroImage: https://res.cloudinary.com/bizzaboprod/image/upload/c_crop,g_custom,f_auto/v1546982777/uj2i17bmkcyfuvxakuem.png
published: true
---

<p class="text-center"><img alt="GopherCon 2019 Banner" src="https://res.cloudinary.com/bizzaboprod/image/upload/w_740,c_fit,fl_any_format,q_auto:best/mjkpm9uwwnknzs4u7b6s.png" class="h5"></p>

Sourcegraph is the standard developer platform for code search and navigation at many of the largest and most exacting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.

For the third year, Sourcegraph is proud to host the official liveblog for GopherCon. We're looking for attendees who want to contribute - is that you?

## Why liveblog

Liveblogging is an excellent way to get more involved with the Go community, support the presenters, and get your name on an artifact of GopherCon. Sign up for multiple sessions to keep your options open and help make sure the schedule gets filled.

## How it works

> Liveblogging is a live stream in written form. It's capturing the essential parts of a talk, converting that into Markdown including images is cool too), then making a pull request against our [website repository](https://github.com/sourcegraph/about). Once your pull request is merged, it will live on the [GopherCon blog](/go) in less than 5 minutes.

Reviewer: Do we have  a liveblog template they can clone? Should also make sure they know what directory/where to put the file, and if we have any guidelines.

```suggestion
Liveblogging is a live stream in written form. It's capturing the essential parts of a talk, and converting that into Markdown (including images is awesome!). Next, make a pull request against our [website repository](https://github.com/sourcegraph/about) to get it hosted on [sourcegraph.com](https://about.sourcegraph.com/blog/). Once your pull request is merged, it will be live on the [GopherCon blog](/go) in less than 5 minutes.
```

> Liveblogging is a live stream in written form. It's capturing the essential parts of a talk, converting that into Markdown including images is cool too), then making a pull request against our [website repository](https://github.com/sourcegraph/about). Once your pull request is merged, it will live on the [GopherCon blog](/go) in less than 5 minutes.

Reviewer: Trying another format:

```suggestion
Liveblogging is a live stream in written form to capture the essential parts of a talk, and convert that into a published blog post. 

Steps to live blog:
- Take notes capturing talk key points.
- Convert notes to Markdown - including images is a bonus!
- Clone the [blog post template](#)
- Make a pull request against our [website repository](https://github.com/sourcegraph/about).
- After your pull request is merged, it will be live on the [GopherCon blog](/go) in less than 5 minutes.
```

We will need to actually link to the template above. 

Is the GopherCon blog url correct?

Author: > Do we have a liveblog template they can clone?

Yes. All of the blog post files will already exist so they just have to fork and add content.

Author: I like what you've done so will commit your suggestion, then re-word to say something along the lines of "find the blog post file for your talk, then edit..."

> We will then promote your post from the  GopherCon and Sourcegraph Twitter accounts, crediting you and raising your profile to over 28,000 developers.

Reviewer: ```suggestion
When your blog post is live, we will promote your post from the GopherCon and Sourcegraph Twitter accounts, crediting you and raising your profile to over 28,000 developers.
```

## To sign-up

[Fill out our Google form](http://bit.ly/gophercon-liveblog-signup-2019) with your contact details and the talks you want to cover. We will then be in touch to confirm your assignment.

## Meet the team at GopherCon 2019!

Our remote-first team from around the world is coming together at GopherCon next week. We are looking forward to seeing our friends in the Go community, so make sure to find us while you're there! You can find a Sourcegrapher by spotting someone wearing one of these hoodies:

![Sourcegraph GopherCon hoodie](images/3.6-gophercon-hoodie.jpg "Sourcegraph GopherCon hoodie")

> And one more thing. Livebloggers will also receive one of these hoodies at the end of the conference.

Reviewer: ```suggestion
One more thing: Livebloggers will receive one of these awesome hoodies at the end of the conference!
```
